prow/config.yaml

```diff
@@ -260,7 +260,7 @@ presubmits:
     trigger: "(?m)^/test pull-kubernetes-e2e-gke-gpu,?(\\s+|$)"
   - name: pull-kubernetes-e2e-kops-aws
     agent: jenkins
-    always_run: true
```

Comment:
I think we still want to run it, we just want it to be non-blocking.

Reply:
(or is that what this does?)

Reply:
Then, removing it from the submit queue config should be enough.

Reply:
I think this looks like a move from presubmits to postsubmits, but I'm not sure what other tweaks are needed

Reply:
Ack.. Will change it.

Reply:
Done.